core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/Invoker.scala

```diff
@@ -220,9 +223,3 @@ trait InvokerServerProvider extends Spi {
   def instance(
     invoker: InvokerCore)(implicit ec: ExecutionContext, actorSystem: ActorSystem, logger: Logging): BasicRasService
 }
-
-object DefaultInvokerServer extends InvokerServerProvider {
```

Comment:
Why is this removed?
This is being used.
https://github.com/apache/openwhisk/blob/master/common/scala/src/main/resources/reference.conf#L30

Reply:
I moved it to core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/DefaultInvokerServer.scala